workspaces/sonarqube/plugins/sonarqube-react/src/hooks/useProjectInfo.ts

```diff
@@ -55,4 +74,4 @@ export const useProjectInfo = (
     }
   }
   return { projectInstance, projectKey };
-};
+}
```

Comment:
```suggestion
};
```

We'll see what the CI says but I wonder if this being removed will make prettier unhappy? 🤔